src/filter.js

```diff
@@ -252,29 +254,60 @@ export class Filter extends CriteriaBuilder {
   }
 
   generateFields(columns, entityName) {
-    let excludeColumns = (this.excludeColumns) ? this.excludeColumns.replace(/\s/g, '').split(',') : [];
+    let paths = [];
 
-    if (this.showIdColumns) {
-      columns.id = 'number';
+    let excludeColumns = splitOnColumnOrIdentity(this.excludeColumns);
+    let includeColumns = splitOnColumnOrIdentity(this.includeColumns);
+
+    if (excludeColumns && includeColumns) {
+      excludeColumns = [];
+
+      logger.warn(`both excludeColumns and includeColumns bindables are defined, ignoring excludeColumns`);
```

Comment:
besides it's much easier. as exclude from the include list does the same as include from what's not excluded